mysql-async/src/main/java/com/github/jasync/sql/db/mysql/MySQLConnection.kt

```diff
@@ -262,8 +262,14 @@ class MySQLConnection @JvmOverloads constructor(
             }
         }
 
-        val sslRequest = SSLRequestMessage(setOfNotNull(
+        val clientFoundRows = System.getProperty("jasync.mysql.CLIENT_FOUND_ROWS") != null
```

Comment:
Thanks for the prompt fix.
So, capability can be enabled with JVM option after this PR.

I think it would be nicer if it detects `mysql-r2dbc` dependency and automatically enable the flag?

Reply:
good idea, I will do something like that on the connection factory.